spec/features/consumer/shopping/checkout_spec.rb

```diff
@@ -119,12 +119,17 @@
         end
       end
 
-      it "doesn't tell about previous orders" do
+      it "shows only applicable content" do
+        # Unifying several specs here to speed up test run.
```

Comment:
IMO it's not worth leaving this comment. I'd have left it as a comment in the PR itself.

Reply:
Okay, I removed them.